lib/src/commands/test/test.dart

```diff
@@ -93,6 +94,12 @@ class TestCommand extends Command<int> {
             'should update the golden files.',
         negatable: false,
       )
+      ..addFlag(
+        'ansi-output',
```

Comment:
Maybe we should rename this flag to `force-ansi` to make it more clear what it does?

Reply:
Agreed

Reply:
@wolfenrain  @felangel: renaming suggestion done

Reply:
@renancaraujo  @felangel  Rebase done juste now. Happy if you are others feedback